### PR TITLE
ReplicatorDispatchRate support cross multiple clusters

### PR DESCRIPTION
**Pull request by suiyuzeng**
Master Issue: #13444

### Motivation
ReplicatorDispatchRate support cross multiple clusters

### Documentation

- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java [hunk 1]
```diff
@@ -85,6 +85,10 @@ public CmdTopicPolicies(Supplier<PulsarAdmin> admin) {
         jcommander.addCommand("get-subscription-dispatch-rate", new GetSubscriptionDispatchRate());
         jcommander.addCommand("set-subscription-dispatch-rate", new SetSubscriptionDispatchRate());
         jcommander.addCommand("remove-subscription-dispatch-rate", new RemoveSubscriptionDispatchRate());
+
+        jcommander.addCommand("get-replicator-dispatch-rate",new GetReplicatorDispatchRate());
```
Comment (yuruguo) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java:89]: A space is required after the comma, from L89 to L91

Comment (suiyuzeng) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java]: fixed

**Review by yuruguo — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**